load_issues.py

```diff
@@ -99,79 +101,150 @@ def parse_issue_template(source_repo: Repository) -> Dict[str, Any]:
+        codeuw_state["last_modified"] = int(time.time())
+    return codeuw_state
 
+def get_state(state_file: str = '.codeuw-state.mpk') -> OrderedDict:
+    """
+    Get the state dictionary from the state file
+    """
+    state_file = Path(state_file)
+    if state_file.exists():
+        # Read the state file when it already exists
+        return read_state(state_file=state_file)
+    else:
+        # If state file doesn't exist, create a new one
+        codeuw_state = OrderedDict({
+            "version": "1.0",
+            "created_time": int(time.time()),
+            "last_modified": int(time.time()),
+            "issues": {}
+        })
+        return codeuw_state
+
+def write_state(state_dict: OrderedDict, state_file: "str | Path" = '.codeuw-state.mpk') -> None:
+    """
+    Write state file to disk as messagepack file format
+    """
+    state_file = Path(state_file)
+    state_file.write_bytes(msgpack.packb(state_dict))
+    
+def read_state(state_file: "str | Path" = '.codeuw-state.mpk') -> OrderedDict:
+    """
+    Read state file from disk as messagepack file format
+    """
+    state_file = Path(state_file)
+    return msgpack.unpackb(state_file.read_bytes(), strict_map_key=False)
 
 def main(config_file: str = ".codeuw-config.yml", dry_run: bool = False):
     """
     Loads "codeuw" labeled issues from Github
     """
-    g = setup_github()
+    gh = setup_github()
     config = load_config(config_file=config_file)
 
-    # Get source repo
-    source_repo = g.get_repo("/".join([config["owner"], config["repo"]]))
-
-    # Get all issue titles from source repo
-    source_issue_titles = [issue.title for issue in source_repo.get_issues(state="all")]
+    # Get codeuw repo
+    codeuw_repo = gh.get_repo("/".join([config["owner"], config["repo"]]))
+    
+    # Get the state dictionary
+    codeuw_state = get_state()
 
     # Get the template dictionary
-    template_dict = parse_issue_template(source_repo)
+    template_dict = parse_issue_template(codeuw_repo)
 
     for repo in config["repos"]:
         repo_path = "/".join([repo["org"], repo["repo"]])
-        gh_repo = g.get_repo(repo_path)
+        # If repo not in state, add it
+        if repo_path not in codeuw_state["issues"]:
+            codeuw_state["issues"][repo_path] = {}
+
+        gh_repo = gh.get_repo(repo_path)
         # Get issues with codeuw label only
         issues_with_label = [
             issue
             for issue in gh_repo.get_issues()
             if "codeuw" in [label.name for label in issue.labels]
         ]
-        generate_code_uw_issues(
+        codeuw_state = generate_code_uw_issues(
             issues_with_label,
             template_dict,
-            repo,
-            source_issue_titles,
+            repo['name'],
+            codeuw_state,
             gh_repo,
-            source_repo,
+            codeuw_repo,
             dry_run,
         )
+    
+    logger.info(f"State content:\n {codeuw_state}")
```

Comment:
hmm, this could log a lot of data over time. Maybe just log the repos tracked and the number of issues for each?